src/date-adapters/date-fns/index.ts

```diff
@@ -26,6 +26,10 @@ import {
 } from 'date-fns';
 import { DateAdapter } from '../date-adapter';
 
+function timezoneOffset(date: Date | number): number {
```

Comment:
Can we add a test here: https://github.com/mattlewis92/calendar-utils/blob/master/test/calendar-utils-dst.spec.ts

For this method in the date-fns + moment date adapters just to be certain the result is the same for both adapters

Reply:
I'm confused about what the test should do. Is it simply passing a date to both adapters and making sure both return the same offset?

Reply:
> I'm confused about what the test should do. Is it simply passing a date to both adapters and making sure both return the same offset?

Yep, exactly that! By putting it in that file, the timezone offset should be `+60` so you can verify it works for both adapters 😄